rewardquest/src/main/java/org/rewardquest/com/pages/LoginPage.java

```diff
@@ -27,36 +23,61 @@
 	@FindBy(how = How.XPATH, using = ".//button[@class='signIn']")
 	private WebElement signIn;
 
+	@FindBy(how = How.XPATH, using = ".//mat-error")
+	WebElement errorMessage;
+
 	/*
 	 * Initialize all the web elements to driver
 	 */
 	public LoginPage(WebDriver driver) {
 		try {
 			this.driver = driver;
 			PageFactory.initElements(driver, this);
-		}
-		catch(Exception e) {
+			wait = new WebDriverWaits();
+		} catch (Exception e) {
 			e.printStackTrace();
 		}
 	}
 
 	/*
-	 * Clicks on DEMO SITES Clicks on Automation practice table Select column

```

Comment:
Demo Sites and Automation Practice table ??

Reply:
Removed Unnecessary things like Demo Sites and Automation Practice table .